### PR TITLE
Miscellaneous changes that were made for 4.2.7 and 4.2.8 for testing/building purposes

### PR DESCRIPTION
**Pull request by jassigill2000**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -1,6 +1,7 @@
 cmake_minimum_required(VERSION 3.5.0 FATAL_ERROR) #CPACK_DEBIAN_<COMPONENT>_PACKAGE_NAME
 
-find_package(IRODS 4.2.5 EXACT REQUIRED)
+find_package(IRODS 4.2.8 EXACT REQUIRED)
```
Comment (trel) [inline CMakeLists.txt:3]: just remove 4.2.8 for now... leave it IRODS REQUIRED

File: irods_consortium_continuous_integration_test_hook.py [hunk 2]
```diff
@@ -315,7 +320,7 @@ def create_json_config_file_for_unit_test():
 
 def install_testing_dependencies():
     irods_python_ci_utilities.subprocess_get_output(['sudo', '-EH', 'pip', 'install', 'unittest-xml-reporting==1.14.0'])
-    add_shortname_to_etc_hosts()
+    #add_shortname_to_etc_hosts()
```
Comment (trel) [inline irods_consortium_continuous_integration_test_hook.py:323]: why not just remove?

File: CMakeLists.txt (hunk 1 above)
Comment (jassigill2000) [reply CMakeLists.txt]: fixed it.

File: irods_consortium_continuous_integration_test_hook.py (hunk 2 above)
Comment (jassigill2000) [reply irods_consortium_continuous_integration_test_hook.py]: removed it

**Comment by trel** [comment]
change first commit message from 'confiure' to 'configure'

**Comment by jassigill2000** [comment]
did it.

**Pull request merged**